draft-ietf-sframe-enc.md

```diff
@@ -34,8 +31,8 @@ author:
     organization: CoSMo Software
     email: sergio.garcia.murillo@cosmosoftware.io
  -
-    ins: R.L. Barnes
-    name: Richard L. Barnes
+    ins: R. Barnes
```

Comment:
I generally avoid "ins", unless xml2rfc is going to do the wrong thing with "name".  It won't in this case.

```suggestion
```